src/test/end-to-end/pages/claimDetails/claimDet.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+const contactUs = require('../../helpers/contactUs.js');
+
+const claimDetailsConfig = require('./claimDetails.json');
+const claimDetailsContentHelper = require('./claimDetailsContentHelper');
+
+module.exports = async function (allClaimDetailsPages) {
+  // eslint-disable-next-line @typescript-eslint/no-this-alias
+  const I = this;
+  if (allClaimDetailsPages) {
+    I.see('What type of discrimination are you');
+    I.see('claiming?');
+    I.see('What is discrimination?');
+    I.see('Select all that apply.');
+    I.see('Age');
+    I.see('Disability');
+    I.see('Gender reassignment');
+    I.see('Marriage or civil partnership');
+    I.see('Pregnancy or maternity');
+    I.see('Race');
+    I.see('Religion or belief');
+    I.see('Sex (including equal pay)');
+    I.see('Sexual orientation');
+
+    I.click(claimDetailsConfig.what_is_discrimination);
+    I.wait(1);
+    await claimDetailsContentHelper.verifyWhatIsDiscrimination();
+
+    I.click(claimDetailsConfig.contact_us);
+    I.wait(1);
+    await contactUs.verifyContactUs();
+
+    I.checkOption(claimDetailsConfig.age);
+    I.checkOption(claimDetailsConfig.disability);
+    I.checkOption(claimDetailsConfig.gender_reassignment);
+    I.checkOption(claimDetailsConfig.marriage_or_civil_partnership);
+    I.checkOption(claimDetailsConfig.pregnancy_or_maternity);
+    I.checkOption(claimDetailsConfig.race);
+    I.checkOption(claimDetailsConfig.religion_or_belief);
+    I.checkOption(claimDetailsConfig.sex);
+    I.checkOption(claimDetailsConfig.sexual_orientation);
+
+    I.click('Save and continue');
+
+    //What happened to you Page.
+    I.see('Describe what happened to you');
+    await claimDetailsContentHelper.verifyThisIsYourOppurtunityToExplain();
+
+    //The various Links on this page...
+    I.see('What to write for discrimination claims');
+    I.see('What to write for dismissal claims');
+    I.see('What to write for whistleblower claims');
+    I.see("What to write for 'other' claims");
+    I.see('Use this box to describe the events around your dispute, or add to your claim by uploading a document');
+    I.see('You have 2500 characters remaining');
+    I.see('Or upload your summary as a separate document (optional)');
+
+    I.click(claimDetailsConfig.what_to_write_for_discrimination_claims);
+    I.wait(1);
```

Comment:
Is the test running slow? do we need to add waits?

Reply:
The Waits are put in for the links to be expanded before the tests proceed...I can try to comment this and run few tests  on them  and see...

Reply:
This has been implemented as well...